src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -96,64 +98,87 @@ function getRouteForCurrentStep(currentStep: TBankAccountStep): ValueOf<typeof R
     }
 }
 
+/**
+ * Returns selected bank account fields based on field names provided.
+ */
+function getFieldsForStep(step: TBankAccountStep): InputID[] {
+    switch (step) {
+        case CONST.BANK_ACCOUNT.STEP.BANK_ACCOUNT:
+            return ['routingNumber', 'accountNumber', 'bankName', 'plaidAccountID', 'plaidAccessToken', 'isSavings'];
```

Comment:
Is there a reason these aren't also CONST?

Reply:
not sure, it was implemented like that before, I've just taken this function out of the component

Reply:
I think this is not a blocker for now, we can add consts in future PRs if we think it would be cleaner